### ISSUE TITLE
[Feature Request][Android] Enable Demo Mode for screenshots

### ISSUE THREAD

**Issue by timrijckaert** [mobile-dev-inc/maestro#1253]
**Is your feature request related to a problem? Please describe.**

When taking screenshots I wish to have the option to enable _demo mode_.
I'm looking for a solution where I can take stable screenshots for screenshot testing purposes, having the status bar icons change would defeat the purpose.

To enable demo mode on Android following ADB commands will come in handy

# Entering and exit demo mode via ADB
## First, to allow demo mode to be used, run:
```
adb shell settings put global sysui_demo_allowed 1
```

## Next, to actually enable demo mode, run:
```
adb shell am broadcast -a com.android.systemui.demo -e command enter
```

## To exit demo mode, run:

```
adb shell am broadcast -a com.android.systemui.demo -e command exit
```

The above commands would be the minimum amount of commands, however more customisation can happen:

### Setting the time

```
adb shell am broadcast -a com.android.systemui.demo -e command clock -e hhmm 1123
```

### To show the wifi strength at max, run the command below. The level can be any value from 0 to 4.

```
adb shell am broadcast -a com.android.systemui.demo -e command network -e wifi show -e level 4
```

### This also works for setting the mobile network strength:

```
adb shell am broadcast -a com.android.systemui.demo -e command network -e mobile show -e level 4
```

More information can be found:
- https://android.googlesource.com/platform/frameworks/base/+/master/packages/SystemUI/docs/demo_mode.md

**Comment by Fishbowler** [comment]
`takeScreenshot` now has a `cropOn` option. I wonder if this meets the need?

**Comment by timrijckaert** [comment]
Status bar heights aren't uniform across Android (notches, cutouts, OEM quirks), so there's no single crop value that holds across the device matrix.
Cropping and demo mode operate at different layers: cropping is destructive post-processing on the bitmap, whereas demo mode controls what actually gets rendered. Demo mode also pins battery, signal, notification icons in one go, which cropping can't address at all.

So long story short no, they seem like 2 different features imho
